ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -493,6 +495,15 @@ else
 	#
 	. ${R}etc/init.d/rc.vehicle_setup
 
+	#
+	# Play the startup tune (if not disabled or there is an error)
+	#
+	param compare CBRK_BUZZER 782090
+	if [ "$?" != "0" -o "$STARTUP_TUNE" != "1" ]
+	then
+		tune_control play -t $STARTUP_TUNE $STARTUP_TUNE_VOLUME
```

Comment:
Should the `tune_control play -t $STARTUP_TUNE` above (~ line 441) be removed?

https://github.com/PX4/PX4-Autopilot/blob/b4c4d3109e802be09994ea99055abec78c4eaa59/ROMFS/px4fmu_common/init.d/rcS#L437-L444

Reply:
Good catch. I removed the first one. And fixed a few more startup things that were not quite right.